components/inspectit-ocelot-configurationserver-ui/src/components/views/configuration/ConfigurationView.js

```diff
@@ -271,14 +271,14 @@ class ConfigurationView extends React.Component {
           visible={this.state.isConfigurationDialogShown}
           onHide={this.hideConfigurationDialog}
           configurationValue={fileContentWithoutFirstLine}
-          fileName={path + name}
+          fileName={`${path}` + `${name}`}
```

Comment:
Added this change so that `path` and `name` will be converted to string when given to the dialog as a prop since until now there where errors in the browser console that the `path` was a `number` but `string` was required.